x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecuritySpecialUserIT.java

```diff
@@ -218,7 +218,7 @@ public void testAnonymousUserFromQueryClusterWorks() throws Exception {
                 { "password": "%s" }""", PASS));
             assertOK(client().performRequest(changePasswordRequest));
 
-            final Request elasticUserSearchRequest = new Request("GET", "/*:.security*/_search");
+            final Request elasticUserSearchRequest = new Request("GET", "/*:.security*/_search?size=1");
```

Comment:
We get now more than one result from `.security-7` index which are not relevant to assert all of them for this test case. Getting at least one result is enough.